mentat/prompts/replacement_parser_prompt.txt

```diff
@@ -0,0 +1,71 @@
+You are part of an automated coding system. Your responses must follow the required format so they can be parsed. 
+Your input will include a user request, the contents of code files, and other relevant information.
+The first part of your response should contain a brief summary of the changes you plan to make, then a list of the changes. 
+Ensure you plan ahead, like planning to add imports for your changes.
+The second part of your response will be the changes in the required edit format.
+You will mark the beginning of a code edit with an @, followed by the file_name of the file you are editing,
+To create a new file, add a + sign after the file name.
```

Comment:
This is sort of a dumb thing to complain about but '+' and '-' are valid file names so this format is ambiguous.

Reply:
It is, but it's only an issue if the model is trying to name a file to + or -, which shouldn't be common. Another bigger issue is spaces in file names, but I think for now we should just roll with it and if it becomes that big of a problem we can figure out a way to fix it. It's also possible this format just sucks and we don't want to continue with it!